Modules/Devices/src/Devices.Domain/Entities/Identities/IdentityDeletionConfiguration.cs

```diff
@@ -1,47 +1,27 @@
 namespace Backbone.Modules.Devices.Domain.Entities.Identities;
 
-public class IdentityDeletionConfiguration
+public record IdentityDeletionConfiguration
 {
-    public static int LengthOfApprovalPeriod => 7;
-    public static int LengthOfGracePeriod => 14;
+    public static IdentityDeletionConfiguration Instance { get; private set; } = new();
 
-    public static GracePeriodNotificationConfiguration GracePeriodNotification1 { get; } = new()
-    {
-        Time = 12
-    };
+    public double LengthOfApprovalPeriodInDays { get; set; } = 7;
+    public double LengthOfGracePeriodInDays { get; set; } = 14;
 
-    public static GracePeriodNotificationConfiguration GracePeriodNotification2 { get; } = new()
-    {
-        Time = 10
-    };
+    public GracePeriodNotificationConfiguration GracePeriodNotification1 { get; set; } = new() { DaysBeforeEndOfGracePeriod = 12 };
```

Comment:
set could be removed

Reply:
It is used in https://github.com/nmshd/backbone/blob/0117ddba65a3d1aa98928e855b3bc18861f4eee8/Modules/Devices/src/Devices.Application/Extensions/IServiceCollectionExtensions.cs#L19 to overwrite the default values via config (the purpose of this PR).

Same goes for the other comments.